service.subtitles.legendastv/addon.xml

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon
+   id="service.subtitles.legendastv"
+   name="Legendas.TV"
+   version="2.4.4"
+   provider-name="gfjardim">
+  <requires>
+    <import addon="xbmc.python" version="2.26.0"/>
+    <import addon="vfs.libarchive" version="1.0.5" />
+  </requires>
+  <extension point="xbmc.subtitle.module" library="service.py" />
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en">Legendas.TV</summary>
+    <description lang="en">Search and download subtitles from Legendas.TV</description>
```

Comment:
en_GB

Reply:
Is there any sort of convention relative to which region to choose? I'm more comfortable with en_US spelling, etc.

Reply:
Default in Kodi is en_GB since Isengard (this is mandatory). If other languages are required, such as en_US you can ofc include them. But by default you are required to provide the default language (en_GB) which is equivalent to old "en"